minet/cli/commands.py

```diff
@@ -457,12 +457,28 @@ def check_dragnet():
                     'description': 'Retrieve metadata about Youtube comments using the API.',
                     'arguments': [
                         {
-                            'name': 'id',
-                            'help': 'Youtube video\'s id.',
+                            'name': 'column',
+                            'help': 'This argument can either take the name of the column containing the video\'s id/url if a file is passed as an argument, or a single youtube video\'s id/url if there is no file '
+                        },
+                        {
+                            'name': 'file',
+                            'help': 'CSV file containing the Youtube videos ids.',
+                            'type': FileType('r'),
+                            'default': sys.stdin,
+                            'nargs': '?'
+                        },
+                        {
+                            'flags': ['-s', '--select'],
+                            'help': 'Columns to include in report (separated by `,`).',
+                            'type': SplitterType()
                         },
                         {
                             'flags': ['-k', '--key'],
                             'help': 'YouTube API Data dashboard API key.'
+                        },
+                        {
+                            'flags': ['-full'],
+                            'help': 'Youtube Api do not always give all the comments, some replies may be ommited. By adding this flag, one make sure to get all comments'
```

Comment:
`YouTube API does not always return every comments as some replies can be omitted. By adding this flag, one ensures to make every needed API call to retrieve all the comments.`